### ISSUE TITLE
您好，关于LST，

### ISSUE THREAD

**Issue by sea-Lee** [songyouwei/ABSA-PyTorch#72]

**Comment by sea-Lee** [comment]
您好，我用您的代码跑了lstm、ian、atae-lstm三个算法，并且训练多次，发现lstm准确率最好是在76.07%，atae-lstm最好的是77.05%,而ian只有73.57%，但是您的论文里ian应该是结果最好的，请问您知道原因吗？
数据集选得都是restaurant，learning_rate=0.01，batch_size=64,dropout=0.1,epoch=300

**Comment by GeneZC** [comment]
尝试learning_rate=0.001

**Comment by sea-Lee** [comment]
抱歉，上面打错了，learning_rate设置的是0.001，就是您代码里推荐的学习率，这个是和学习率有关吗？那我设置成bert推荐的学习率试试2e-5

**Comment by sea-Lee** [comment]
谢谢您的回答。改变学习率确实能一定程度解决这个问题，当learning_rate: 1e-04，acc=75.98%；learning_rate: 5e-05，acc=76.52%。但代码推荐的学习率是0.001，而代码作者在他论文里的正确率是78.6%，我在想是不是其他一些地方有问题，毕竟5e-05的学习率太小了。
还有个问题是我发现数据集里有的数据标签感觉不是很对。比如：while there 's a decent menu , it should n't take ten minutes to get your drinks and 45 for a dessert pizza .
menu
1
drinks
0
dessert pizza
0
我感觉这里后两个aspect的情感倾向是消极的

**Comment by GeneZC** [comment]
也有可能和dropout有关系

**Comment by minmummax** [comment]
> 您好，我用您的代码跑了lstm、ian、atae-lstm三个算法，并且训练多次，发现lstm准确率最好是在76.07%，atae-lstm最好的是77.05%,而ian只有73.57%，但是您的论文里ian应该是结果最好的，请问您知道原因吗？
> 数据集选得都是restaurant，learning_rate=0.001，batch_size=64,dropout=0.1,epoch=300

您好，我用您的这些参数跑lstm， loss一直在0.90左右 ， 最后的正确率只有 0.65左右 这个是为什么？

**Comment by yfqu96** [comment]
> 数据集选得都是restaurant，learning_rate=0.001，batch_size=64,dropout=0.1,epoch=300

确实是l2正则太大了，多谢
